docs/install-and-run.md

```diff
@@ -3,115 +3,126 @@
 This document shows you how to install EarlGrey and then how to set up and run your first
 test.
 
-## Prerequisites
-
-For EarlGrey to function properly, make sure the app under test has correct launch screen images present for all supported devices (see [iOS Developer Library, Launch Files](https://developer.apple.com/library/ios/documentation/UserExperience/Conceptual/MobileHIG/LaunchImages.html)).
-
 ## Install EarlGrey
 
-You can add EarlGrey to Xcode projects in two ways: using CocoaPods, or as a framework. If you wish to contribute to the EarlGrey project, the **EarlGrey.xcodeproj** generated from the [GitHub Installation](#github-installation) section is integrated with the Unit and Functional Test projects for you to work with the source.
+You can add EarlGrey to Xcode projects in three ways: using CocoaPods, Carthage or manually through the Xcode Project.
+
+For EarlGrey, we highly recommend [CocoaPods](http://cocoapods.org/pods/EarLGrey) as the best way to get started.
 
 ### CocoaPods installation
 
-For EarlGrey, we highly recommend [CocoaPods](https://cocoapods.org/) as the best way to get started.
+#### Step 1: Set up a Unit target      ///// EarlGrey adds changes to the test target's Scheme and Build Phases
 
-#### Step 1: Set up a test target
+1. To create a new Unit Test (XCTest) target, select your project in the Xcode Project Navigator,
+and then click **Editor → Add Target...** from the menu.
+2. In the **Add Target** window, select **iOS** → **Test** → **iOS Unit Testing Bundle**:
 
-  1. EarlGrey requires a **Test Target**. Because EarlGrey adds changes to the test target's Scheme and Build
-  Phases, we recommend that you create a separate test target for adding EarlGrey tests. If you do not have
-  one set up already, you can do so by selecting your project in the Xcode Project Navigator, and then and
-  clicking **Editor → Add Target...** from the menu.
-  2. In the **Add Target** window, select **iOS** → **Test** → **iOS Unit Testing Bundle**:
+<img src="images/image00.png" width="500">
 
-      <img src="images/image00.png" width="500">
+Click Next → "Add a Test Target Name" → Finish.
```

Comment:
make this bold like others? or keep em consistent. also what's special about "Add a Test Target Name". I don't think Xcode 8 has those screens.